bin/start-func-test-server.js

```diff
@@ -32,22 +32,26 @@ function shell(cmd, args) {
       cwd: root,
     });
 
-    child.stdout.on('data', (data) => {
```

Comment:
I wish this part was configurable :(

Reply:
personally, I really hate having to type parentheses all the time :) It's very common to make an arrow function with only one param. We have lint rules to catch mistakes.

Reply:
I value consistency and scan-ability--using parens all the time makes it easier to pick out a function as they always look the same (this is why I dislike implicit return functions as well). We spend more time reading code than writing it, so I think the effort to type something matters less than the effort for a human to scan/parse/read it. 😄

Reply:
It just takes a little getting used to. When your eye sees the arrow (`=>`) it immediately knows what's going on.

Reply:
While the `=>` is a bit of a unique set of characters I definitely look for more than just `=>` when looking for a function. I see parens `()` as a reference to something callable.

My main issue though is having to train my brain and other developers to learn two ways to do the same thing. This is why I also dislike implicit returns: rather than always writing a function as `(param) => { return value; }` I need to write it in a different way depending on context or my style choices or whatever. And developers need to learn that rule. Or prettier would transform their effort of writing the most expressive and clear version to something condensed because it didn't like the verbosity used in this case. And if you're using object de-structuring (which is sort of just using one argument in theory, right?) you still need parens.

I think it's a code smell to have multiple ways of writing the same thing which is why I advocate for always using full function syntax. I think having prettier or whatever always convert to full functions would be way better. Anyone new to the code wouldn't have to learn/wonder why things are written different ways in different places. The whole reason for code style is consistency is key 😄